test/e2e/config/caren.yaml

```diff
@@ -134,8 +134,8 @@ providers:
 - name: caren
   type: RuntimeExtensionProvider
   versions:
-  - name: "{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.13}"
-    value: "https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/releases/download/{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.13}/runtime-extension-components.yaml"
+  - name: "{go://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix@v0.14}"
```

Comment:
I changed it based on documentation at https://nutanix-cloud-native.github.io/cluster-api-runtime-extensions-nutanix/contributing/releasing/ . I am not sure if it was kept at 0.13 intentionally or not.

Reply:
As mentioned on other PR, this is only applicable for upgrade tests so let's figure that out once we add those tests. Happy to merge this as-is anyway because it doesn't affect anything right now.